lms/djangoapps/verify_student/models.py

```diff
@@ -1103,13 +1104,13 @@ def invalidate_deadline_caches(sender, **kwargs):  # pylint: disable=unused-argu
     cache.delete(VerificationDeadline.ALL_DEADLINES_CACHE_KEY)
 
 
-class VerificationCheckpoint(models.Model):
-    """Represents a point at which a user is asked to re-verify his/her
-    identity.
-
-    Each checkpoint is uniquely identified by a
-    (course_id, checkpoint_location) tuple.
+class VerificationCheckpoint(DeprecatedModel):  # pylint: disable=model-missing-unicode
```

Comment:
As it turns out, `model-no-explicit-unicode` is *also* a pylint failure 😢. Also, this latest commit works correctly without needing to specify the child `__init__` but pylint still doesn't like it.

https://build.testeng.edx.org/job/edx-platform-quality-pr/31098/Diff_Quality_Report/